pkg/dashboard/ui/src/app/shared/services/nuclio-functions-data.service.js

```diff
@@ -23,9 +23,10 @@
          * Update existing function with new data
          * @param {Object} functionDetails
          * @param {string} projectName - the name of the project containing the function
+         * @param {boolean} importProcess - "true" if importing process
```

Comment:
When we have an optional parameter we document it in JSDoc by putting its name inside brackets `[]`.
If it has a default value we also add a `=` then the default value.
```suggestion
         * @param {boolean} [importProcess=false] - "true" if importing process
```